cart/world.go

```diff
@@ -105,6 +128,22 @@ func (world *World) Update(t int32, player *Player, game *Game) {
 ///////////////////////////////////////////////////////////////////////////////
 // Utils
 
+func treeLifeDetails(life int8) (byte, byte, byte) {
+	switch life {
+	case 3:
+		return 16, 31, 128
+	case 2:
+		return 144, 159, 16
+	case 1:
+		return 160, 175, 16
+	case 0:
+		return 176, 191, 0
```

Comment:
We can update this to give the tree a longer life

Reply:
You can have named return variables in Go with
```go
func foo() (a int32, b int32, c int32) {
    a = 10
    b = 20
    c = 30
    return
}
```
Can we maybe use that here just to give a bit more meaning to the tripple?

Reply:
Done